R/plot_carp.R

```diff
@@ -32,38 +32,39 @@
-#'        other plot types) - must be positive.
-#' @param dend.labels.cex Label size for dendrogram plots (ignored for other plot
-#'        types) - must be positive.
-#' @param dend.ylab.cex Y-axis size for dendrogram plots (ignored for other plot
-#'        types) - must be positive.
+#'            \item when \code{type == "heatmap"} and \code{interactive == TRUE},
+#'                  \code{...} is forwarded to \code{\link[heatmaply]{heatmaply}}.
+#'            } See the documentation of \code{\link[ggplot2]{ggplot}} (for \code{interactive == FALSE})
+#'            or \code{\link[plotly]{plotly}} (for \code{interactive == TRUE}) for details about additional
+#'            supported arguments to the corresponding plot \code{type}.
 #' @param slider_y A number to adjust the slider's vertical position for
 #'                 interactive dendrogram and interactive heatmap plots
 #'                 (ignored for other plot types).
-#' @return The value of the return type depends on the \code{type} argument:\itemize{
-#'   \item if \code{type = "dendrogram"}, \code{x} is returned invisibly;
-#'   \item if \code{type = "path"}, an object of class \code{\link[ggplot2]{ggplot}}
+#' @param refit A logical scalar. Should "naive" centroids (TRUE) or the
+#'              actual centroids estimated by convex clustering be used?
+#'              When the default refit = FALSE, the estimated U from the convex
+#'              clustering problem is used. The refit = TRUE returns actual
+#'              centroids (mean) of all elements assigned to that cluster;
+#'              Due to the global shrinkage imposed, these clusters are
+#'              more "shrunk together" than the naive clusters. Only for the
+#'              heatmap plots. (ignored for other plot types).
+#' @return The value of the return type depends on the \code{interactive} and \code{dynamic} arguments:\itemize{
+#'   \item if \code{interactive = FALSE} and \code{dynamic = FALSE}, an object of class \code{\link[ggplot2]{ggplot}}
 #'         which can be plotted directly (by invoking its print method) or modified
 #'         further by the user is returned;
-#'   \item if \code{type = "dynamic_path"}, an object of class \code{\link[gganimate:gganimate-package]{gganim}}
+#'   \item if \code{interactive = FALSE} and \code{dynamic = TRUE}, an object of class \code{\link[gganimate:gganimate-package]{gganim}}
 #'         is returned, and many be further manipulated by the user or plotted directly;
-#' } \code{saveviz.CARP} always returns \code{file.name} invisibly.
-#' @details The \code{\link{saveviz.CARP}} function provides a unified interface
-#'          for exporting \code{CARP} visualizations to files. For all plots,
-#'          at most one of \code{percent} and \code{k} may be supplied.
+#'   \item if \code{interactive = TRUE}, an object of class \code{\link[plotly]{plotly}}
+#'         is returned, and many be further manipulated by the user or plotted directly.
```

Comment:
Maybe move this comment about "further manipulated" outside the bulleted list since all objects can be manipulated?